src/util/misc.rs

```diff
@@ -335,9 +338,14 @@ mod tests {
         let p2pkh = ::Address::p2pkh(&pubkey, ::Network::Bitcoin);
```

Comment:
Not relevant to this PR but I noticed we don't have `Ok(false)` test for broken signatures.

Reply:
@Kixunil I created https://github.com/rust-bitcoin/rust-bitcoin/pull/825 to test broken signatures